### PR TITLE
Add Avatar component

### PR DESCRIPTION
**Pull request by nikhilmalviya-dev**

### CODE REVIEW

File: Avatar/avatar.css [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+#ambani3{
+    width: 50px;
+    height: 50px;
+    border-radius:50%
+}
+#ambani4{
+    width: 40px;
+    height: 40px;
+    border-radius:50%
+}
```
Comment (devananth) [inline Avatar/avatar.css]: I would suggest not use ID , keep a generic class for different sizes. `avatar-sm' , `avatar-md`.

Avoid using pixels , use rems

`border-radius:50%` is repeating , you could keep a generic class for that

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,3 +1,3 @@
 {
-    "liveServer.settings.port": 5502
+    "liveServer.settings.port": 5503
```
Comment (junaidshaikh-js) [inline .vscode/settings.json]: Delete this file or add it to gitignore file.

File: Avatar/avatar.css [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+    background-color:rgb(139, 185, 179);
+    color:black;
+    font-weight: bold;
+    font-size:19px;
+}
+.home{
+    display: flex;
+    gap:0.5rem;
+}
+#home-link{
```
Comment (junaidshaikh-js) [inline Avatar/avatar.css]: id should not be used for styling

File: Avatar/avatar.css [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+}
+.componant{
+    text-decoration: none;
+    color:grey;
+    font-weight: bold;
+    font-size: 20px;
+}
+.componant:hover{
+    text-decoration: underline;
+    color: black;
```
Comment (junaidshaikh-js) [inline Avatar/avatar.css:63]: use variable if values gets repeated

File: Avatar/avatar.css [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+#ambani1{
+    width:100px;
+    height: 100px;
+    border-radius:50%
+}
+#ambani2{
+    width:75px;
+    height: 75px;
+    border-radius:50%
+
```
Comment (junaidshaikh-js) [inline Avatar/avatar.css]: remove blank lines from code

**Review by junaidshaikh-js — COMMENTED** [review]
Good Work! 

Add description about your PR, and don't use ids for styling, it is not recommended.

File: Avatar/avatar.css (hunk 1 above)
Comment (nikhilmalviya-dev) [reply Avatar/avatar.css]: okk will do that.

File: .vscode/settings.json (hunk 2 above)
Comment (nikhilmalviya-dev) [reply .vscode/settings.json]: ok thanks for your suggestion

**Comment by devananth** [comment]
you can merge it .

**Review by DeshmukhAkash2000 — APPROVED** [review]

**Review by Bharatrathod-81 — APPROVED** [review]
seems good you can go for merge

**Pull request merged**